app/views/users/_user.html.erb

```diff
@@ -0,0 +1,8 @@
+<li>
+  <%= gravatar_for user, size: Settings.index.size %>
+  <%= link_to user.name, user %>
+  <% if current_user.admin? && !current_user?(user) %>
+    | <%= link_to t(".del"), user, method: :delete,
+    data: { confirm: t(".confirm") } %>
```

Comment:
remove the white space after beginning and before ending of curly brace

Reply:
Yes